tfhe/benches/integer/bench.rs

```diff
@@ -1400,6 +1404,121 @@ mod cuda {
         bench_group.finish()
     }
 
+    fn cuda_compress(c: &mut Criterion) {
+        let bench_name = "integer::cuda::compression";
```

Comment:
never know what format the bench names need to have, I'm assuming you followed the existing convetion ?

Reply:
I believe so. Other integer operations go as `integer::cuda::(un)signed::" , but since compression is the same for signed and unsigned I believe we can omit that.